libs/components/indicators/src/lib/modules/help-inline/help-inline.component.ts

```diff
@@ -25,6 +25,7 @@ export class SkyHelpInlineComponent {
   /**
    * The ARIA label for help inline button. This sets the button's `aria-label` to provide a text equivalent for screen readers
    * [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility).
+   * When using multiple help inline components on a single page, each one should be given a unique ARIA label so that users can differentiate which help element is currently in focus.
```

Comment:
```suggestion
   * When you use multiple help inline components on a page, give each one a unique ARIA label so that users can differentiate which help element is currently in focus.
```